packages/strapi-design-system/src/Popover/Popover.js

```diff
@@ -44,7 +45,7 @@ const PopoverScrollable = styled(Box)`
   }
 `;
 
-export const Popover = ({ source, visible, children, ...props }) => {
+export const Popover = ({ source, visible, children, spacing, ...props }) => {
```

Comment:
That spacing prop can be a bit confusing. As a developer, I expect to have spacing on every side, not just the top.

Reply:
Sounds good 😊 what could be another valid name?

Reply:
I don't really know maybe something like `spacingTop`, `verticalSpacing` or `marginTop`.

Reply:
I've taken spacingTop